middleware/markdown/summary/summary.go

```diff
@@ -0,0 +1,16 @@
+package summary
```

Comment:
Actually I think excerpt is a pretty good word for it. But we can merge the PR in either way, and change it later.